apps/common-app/src/examples/AudioFile/AudioFile.tsx

```diff
@@ -61,15 +61,15 @@ const AudioFile: FC = () => {
         fetchAudioBuffer();
       }
 
-      bufferSourceRef.current = audioContextRef.current.createBufferSource();
+      bufferSourceRef.current =
+        audioContextRef.current.createBufferSource(true);
```

Comment:
Almost there, can you make it an options object?

```tsx
audioContextRef.current.createBufferSource({ pitchCorrection: true });
```

it will be consistent with AudioContext creation